recipe/meta.yaml

```
@@ -57,6 +57,10 @@ build:
   # 2024/04 This build seems to fail due to disk space on the CIs
   # We can potentially re-enable it at a later time
   skip: true  # [cuda_compiler_version == "11.8" and aarch64]
+  skip: true  # [py!=310]
+  skip: true  # [osx]
+  skip: true  # [aarch64]
+  skip: true  # [linux64]
```

```suggestion
  skip: true  # [linux64]
  skip: true  # [cuda_compiler_version != "None"]
```

While you are hitting failures in the CPU build, it would be nice to just cancel the cuda builds too ;)